docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -7,22 +7,20 @@ FileUploaderComponentsExample, LevelExample, IsPreviewerVisibleExample, Variatio
 MaxSizeExample, MultipleExample, ResumableExample, ShowImagesExample, FileUploaderSuccessExample, PreviewerComponentsExample, TrackerComponentsExample, UploadButtonComponentsExample } from './examples'
 import { ReactPropsTable } from '../../in-app-messaging/props'
 
-<Alert role="none" heading="Wait!">
-  Did you follow the [quick start instructions](/connected-components/storage#quick-start) to set up the storage and auth services?
+<Alert variation="info" heading="Developer Preview">
```

Comment:
Git diff is weird here, moved the dev preview alert to the top, and the setup alert to under the Overview section.